src/functions.js

```diff
@@ -100,6 +100,9 @@ module.exports = function(codegen) {
         throw new Error('Too many arguments to if function.');
         var a = args.map(codegen);
         return a[0]+'?'+a[1]+':'+a[2];
-      }
+      },
+
+    // functions defined in function_definitions
+    'inRange': 'fn.in_range'
```

Comment:
Yeah, this should be inRange. I changed it to match the conventions of the other functions. I'll fix it